### ISSUE TITLE
Inconsistency between DB and streaming commands

### ISSUE THREAD

**Issue by ddieruf** [datastax/astra-cli#3]
This may be a deeper issue than the CLI, but I'll start here. I have found a few inconsistencies between commands. Specifically the --cloud and --region parameters.

To create a DB in GCP us central 1:
`astra db create --keyspace my-keyspace --cloud "GCP" --region "us-central1" a-db`

To create a tenant in GCP us central 1:
`astra streaming create --cloud "gcp" --region "uscentral1" a-tenant`

It would be nice for the cloud and region values to be the same for both.

**Comment by clun** [comment]
Maintaining a mapping table on the CLI side could help but would be hard to maintain.

I would propose more lookup commands for user to know which regions he can use

```
astra db regions list-serverless [--cloud aws]
astra db regions list-classic  [--cloud aws]
astra streaming regions list [--cloud aws]
```

**Comment by ddieruf** [comment]
This is good. Then, at least someone has more clarity into regions. The region names are case-sensitive in the Astra api right?

**Comment by ddieruf** [comment]
So I can do `astra streaming regions list --cloud aws --output json` and figure out the region. Then treat it as an object and not worry about case.

**Comment by clun** [comment]
First implementation fo db classic

> `astra  org list-regions-db-classic`

```
+----------------+-------------------------+-------------------------------+
| Cloud Provider | Region                  | Full Name                     |
+----------------+-------------------------+-------------------------------+
| aws            | af-south-1              | Africa (Cape Town)            |
| aws            | ap-east-1               | Hong Kong                     |
| aws            | ap-south-1              | Asia Pacific (Mumbai)         |
| aws            | ap-southeast-1          | Asia Pacific (Singapore)      |
| aws            | ap-southeast-2          | Asia Pacific (Sydney)         |
| aws            | eu-central-1            | Europe (Frankfurt)            |
| aws            | eu-west-1               | EU West 1 (Ireland)           |
| aws            | sa-east-1               | Sao Paulo                     |
| aws            | us-east-1               | US East (N. Virginia)         |
| aws            | us-east-2               | US East (Ohio)                |
| aws            | us-west-2               | US West (Oregon)              |
| azure          | australiaeast           | Australia East                |
| azure          | brazilsouth             | Brazil South                  |
| azure          | canadacentral           | Canada Central (Toronto)      |
| azure          | centralindia            | Central India                 |
| azure          | eastus2                 | East US 2 (Virginia)          |
| azure          | northeurope             | North Europe                  |
| azure          | southcentralus          | South Central US              |
| azure          | westeurope              | West Europe                   |
| azure          | westus2                 | West US 2 (Washington)        |
| gcp            | asia-south1             | Mumbai, India                 |
| gcp            | europe-west1            | St. Ghislain, Belgium         |
| gcp            | northamerica-northeast1 | NA Northeast 1 (Montreal)     |
| gcp            | southamerica-east1      | Osasco, Sao Paulo, Brazil     |
| gcp            | us-central1             | US Central (Iowa)             |
| gcp            | us-east1                | US East (South Carolina)      |
| gcp            | us-east4                | US East 4 (Virginia)          |
| gcp            | us-west1                | The Dalles, Oregon            |
+----------------+-------------------------+-------------------------------+
```

> `astra  org list-regions-db-classic --cloud aws`

```
+----------------+---------------------+-------------------------------+
| Cloud Provider | Region              | Full Name                     |
+----------------+---------------------+-------------------------------+
| aws            | af-south-1          | Africa (Cape Town)            |
| aws            | ap-east-1           | Hong Kong                     |
| aws            | ap-south-1          | Asia Pacific (Mumbai)         |
| aws            | ap-southeast-1      | Asia Pacific (Singapore)      |
| aws            | ap-southeast-2      | Asia Pacific (Sydney)         |
| aws            | eu-central-1        | Europe (Frankfurt)            |
| aws            | eu-west-1           | EU West 1 (Ireland)           |
| aws            | sa-east-1           | Sao Paulo                     |
| aws            | us-east-1           | US East (N. Virginia)         |
| aws            | us-east-2           | US East (Ohio)                |
| aws            | us-west-2           | US West (Oregon)              |
+----------------+---------------------+-------------------------------+
```

> `astra  org list-regions-db-classic --cloud aws --filter us`

```
+----------------+---------------------+-------------------------------+
| Cloud Provider | Region              | Full Name                     |
+----------------+---------------------+-------------------------------+
| aws            | us-east-1           | US East (N. Virginia)         |
| aws            | us-east-2           | US East (Ohio)                |
| aws            | us-west-2           | US West (Oregon)              |
+----------------+---------------------+-------------------------------+
```

**Comment by clun** [comment]
Feature is available in v0.1.alpha6
